controllers/services/monitoring/monitoring_controller.go

```diff
@@ -69,29 +42,13 @@ func NewServiceReconciler(ctx context.Context, mgr ctrl.Manager) error {
 		//   for to objects that have the label components.platform.opendatahub.io/part-of
 		// or services.platform.opendatahub.io/part-of set to the current owner
 		//
-		Watches(&extv1.CustomResourceDefinition{}).
+		Watches(&dscv1.DataScienceCluster{}, reconciler.WithEventHandler(handlers.ToNamed(serviceApi.MonitoringInstanceName)),
```

Comment:
do we want to watch monitoring CR as well?

Reply:
also for the
Owns(&corev1.PersistentVolumeClaim{}).
Owns(&monitoringv1.PrometheusRule{}). 
should be kept?

Reply:
I think we will be watching the monitoring CR by default.

the other 2 resources are being watched by Dsci